MapboxNavigation/RouteMapViewController.swift

```diff
@@ -984,10 +984,12 @@ extension RouteMapViewController {
     @objc fileprivate func keyboardWillShow(notification: NSNotification) {
         guard navigationView.endOfRouteView != nil else { return }
         guard let userInfo = notification.userInfo else { return }
-        let curve = UIViewAnimationCurve(rawValue: userInfo[UIKeyboardAnimationCurveUserInfoKey] as! Int)
-        let options = (duration: userInfo[UIKeyboardAnimationDurationUserInfoKey] as! Double,
-                       curve: curve!)
```

Comment:
This `!` is the problem, per https://github.com/mapbox/mapbox-navigation-ios/issues/1600#issuecomment-411948817. Let’s whittle this PR down to just handling the case where `curve` is `nil`.